Felienne/hedy
# [BUG FIX] Language switchers correctly aligns on RTL languages

**Description**
In this PR we fix an issue where the language switcher to translate the code was always aligned in the top-right corner of the main editor. However, for the RTL languages this alignment should be on the left-top corner. We fix this by adjusting the Tailwind classes on the dropdown-menu to make sure the switcher is always in the correct spot.

**Fixes**
This PR fixes #2062

**How to test**
Switch from an LTR to an RTL language, for example Dutch to Arabic and notice that the language switcher correctly aligns to the correct top-corner of the code editor.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
